assets/js/base/components/validation/style.scss

```diff
@@ -0,0 +1,6 @@
+.wc-block-form-input-validation-error {
+	color: $error-red;
+	@include font-size(12);
+	padding-left: $gap-small;
```

Comment:
Currently it looks a bit misaligned, the error notice left padding size is between the border and the input text:
![imatge](https://user-images.githubusercontent.com/3616980/76757662-42abf400-6788-11ea-9e42-b12e129db12a.png)
I would either remove the `padding-left` completely, or make it like it is in the text input CSS (`$gap` ).

Reply:
Agreed, and fixed.